SemanticSearchV2.md

```diff
@@ -163,20 +164,17 @@ https://api.careerbuilder.com/core/semanticsearch/document
 
 # Document Response
 
-The document response is divided into two parts. First is the parsed_input node which contains information on all extracted keywords and phrases. Next is the extracted_keywords node, which gives the name, type and relationships of all extracted phrases:
+The document response includes the following parts. First is the `parsed_input` node which contains information on all extracted keywords and phrases. Next is the `semantic_keywords` node, which gives the name, type and relationships of all extracted phrases. Also, `job_titles`, `location`, `geography` and etc json objects may also be present. These objects are derived from parsing the job or resume document, and may not appear when the supplied document lacks corresponding information. 
```

Comment:
Instead of "Also, `job_titles`, `location`, `geography` and etc json objects may also be present." I would say "The response may also include additional objects such as, `job_titles`, `location`, and `geography`.